cob_controller_configuration_gazebo/src/gazebo_services.py

```diff
@@ -12,11 +12,11 @@
 class gazebo_services():
 
 	def __init__(self):
-		self.action_client = actionlib.SimpleActionClient('follow_joint_trajectory', FollowJointTrajectoryAction)
-		self.init_srv = rospy.Service('init', Trigger, self.init_cb)
-		self.stop_srv = rospy.Service('stop', Trigger, self.stop_cb)
-		self.recover_srv = rospy.Service('recover', Trigger, self.recover_cb)
-		self.set_operation_mode_srv = rospy.Service('set_operation_mode', SetOperationMode, self.set_operation_mode_cb)
+		self.action_client = actionlib.SimpleActionClient('joint_trajectory_controller/follow_joint_trajectory', FollowJointTrajectoryAction)
```

Comment:
we do not need the actionclient and the stop service any more because there is no more stop service offered by hardware. The script server will directly cancel the follow_joint_trajectory action.

there is no more set_operation_mode service. --> delete

there is new services halt and shutdown. --> add